### PR TITLE
Added a collaborators page which shows github contributors

### PR DESCRIPTION
**Pull request by LionelBergen**
https://github.com/api-platform/website/issues/60

### CODE REVIEW

**Comment by lex111** [comment]
@LionelBergen please fix [founded errors](https://travis-ci.org/api-platform/website/builds/595815086?utm_source=github_status&utm_medium=notification).

**Comment by LionelBergen** [comment]
@lex111  Should be okay now

File: src/lib/apiHelper.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+function getRepositoriesList() {
+  return axios.get(encodeURI(GITHUB_GET_API_PLATFORM_REPOS_URL)).then(response => {
+    return response.data
+      .map(e => getContributorLinkFromResponse(e))
+      .filter(e => !REPOSITORIES_TO_IGNORE.includes(e.name));
+  });
+}
+
+function getListOfContrubotrsFromGithubApiLinks(listOfContributorsApiLinks) {
```
Comment (dunglas) [inline src/lib/apiHelper.js]:
```suggestion
function getListOfContributorsFromGithubApiLinks(listOfContributorsApiLinks) {
```

:)

**Comment by dunglas** [comment]
Thanks for this contribution! I just tested it, and it works, but unfortunately only one time, then I get this error `API rate limit exceeded for 188.165.96.186. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)`.

It's because de computation is done client-side, wouldn't it be better to do it server-side? For instance we could compute the stats when deploying the docs, and then always use the computed data?

Also, there are so visual glitches (especially on the homepage).

**Comment by LionelBergen** [comment]
Alright I'll update it to server-side. Are we okay with the information only being updated whenever the docs are deployed?

**Comment by dunglas** [comment]
Yes, but I think it's the better solution we have right now. Fortunately, the website is deployed every time we merge a docs PR, and it happens often.

**Comment by dunglas** [comment]
Also, could you remove `dependabot` from the list of contributors, it's not a real one :)

**Comment by alanpoulain** [comment]
Supersed by https://github.com/api-platform/website/pull/191.

**Pull request closed without merging**